ecs-init/config/common.go

```diff
@@ -147,3 +147,11 @@ func DockerUnixSocket() (string, bool) {
 func CgroupMountpoint() string {
 	return cgroupMountpoint
 }
+
+// HostCertsDirPath() returns the CA store path on the host
+func HostCertsDirPath() string {
+	if _, err := os.Stat(hostCertsDirPath); os.IsNotExist(err) {
```

Comment:
Any reason not to just use `err != nil` here?

Reply:
just chatted with @jahkeup about this, you're right we can just capture all errors.